apps/mobile/src/components/DeepLinkRegistrar.tsx

```diff
@@ -45,6 +45,22 @@ export function DeepLinkRegistrar() {
       const parsedUrl = new URL(url);
       const splitBySlash = parsedUrl.pathname.split('/').filter(Boolean);
 
+      if (parsedUrl.pathname.includes('post/')) {
```

Comment:
Added `/` to validate, just in case if there is an username that has `post` inside it